x-pack/plugins/maps/common/constants.ts

```diff
@@ -41,6 +41,7 @@ export const GIS_API_PATH = `api/${APP_ID}`;
 export const INDEX_SETTINGS_API_PATH = `${GIS_API_PATH}/indexSettings`;
 export const FONTS_API_PATH = `${GIS_API_PATH}/fonts`;
 export const INDEX_SOURCE_API_PATH = `${GIS_API_PATH}/docSource`;
+export const INDEX_FEATURE_PATH = `/${GIS_API_PATH}/feature`;
```

Comment:
I know the route URL did not change in this PR but `feature` is extremely vague and provides no indication of what this route does. How about changing the the route path to be more descriptive, something like "add_features_to_index"?

Reply:
We're trying to strip this PR down from the original POC to just be focused on index & index pattern creation, so would prefer not to do any major mods to the feature endpoint at this point.

All that said, this was intentionally left vague to potentially handle other operations besides just adding. We're definitely going to be doing more work on the routes in terms of what requests are handled (i.e.- update, delete). Let's have some discussion offline to talk about what we want these routes to look like moving forward then we can potentially tackle route additions, updates and/or modifications in a separate dedicated PR.